tests/performance/test_performance.py

```diff
@@ -0,0 +1,779 @@
+
+            owner = groups[random.randint(1, len(groups) - 1)][0].display_name
+            transfer_ownership(sql_backend, "TABLE", full_name, owner)
+            for group, permissions in table_permission.items():
+                for permission in permissions:
+                    to_persist.append(ObjectPermission(group, "TABLE", full_name, permission))
+            to_persist.append(ObjectPermission(owner, "TABLE", full_name, "OWN"))
+
+        for j in range(nb_of_tables):
+            view = make_table(schema_name=schema.name, view=True, ctas="SELECT 2+2 AS four")
+            full_name = schema.name + "." + view.name
+            view_permission = create_hive_metastore_permissions(groups, ["SELECT"])
+            grant_permissions(sql_backend, "VIEW", full_name, view_permission)
+
+            owner = groups[random.randint(1, len(groups) - 1)][0].display_name
+            transfer_ownership(sql_backend, "VIEW", full_name, owner)
+            for group, permissions in view_permission.items():
+                for permission in permissions:
+                    to_persist.append(ObjectPermission(group, "VIEW", full_name, permission))
+            to_persist.append(ObjectPermission(owner, "VIEW", full_name, "OWN"))
+        for j in range(nb_of_tables):
+            table = make_table(schema_name=schema.name)
+            full_name = schema.name + "." + table.name
+            table_permission = create_hive_metastore_permissions(groups, ["SELECT", "MODIFY", "READ_METADATA"])
+
+            grant_permissions(sql_backend, "TABLE", full_name, table_permission)
+
+            owner = users[random.randint(1, len(users) - 1)].display_name
+            transfer_ownership(sql_backend, "TABLE", full_name, owner)
+            for group, permissions in table_permission.items():
+                for permission in permissions:
+                    to_persist.append(ObjectPermission(group, "TABLE", full_name, permission))
+
+            to_persist.append(ObjectPermission(owner, "TABLE", full_name, "OWN"))
+        for j in range(nb_of_tables):
+            view = make_table(schema_name=schema.name, view=True, ctas="SELECT 2+2 AS four")
+            full_name = schema.name + "." + view.name
+            view_permission = create_hive_metastore_permissions(groups, ["SELECT"])
+
+            grant_permissions(sql_backend, "VIEW", full_name, view_permission)
+
+            owner = users[random.randint(1, len(users) - 1)].display_name
+            transfer_ownership(sql_backend, "VIEW", full_name, owner)
+            for group, permissions in view_permission.items():
+                for permission in permissions:
+                    to_persist.append(ObjectPermission(group, "VIEW", full_name, permission))
+            to_persist.append(ObjectPermission(owner, "VIEW", full_name, "OWN"))
+    sql_backend.save_table(f"{test_database.name}.objects", to_persist, ObjectPermission)
+
```

Comment:
Wouldn't it conflict with the assessment job?

Reply:
It's not the same database than the assessment job, the variable name isn't quite right